src/mon/MonClient.h

```diff
@@ -144,8 +144,52 @@ class MonConnection {
   uint64_t global_id;
 };
 
+const boost::system::error_category& monc_category() noexcept;
+
+namespace monc_errc {
+enum monc_errc_t {
```

Comment:
Any reason to not prefer a scoped enum?

Reply:
No. I had thought boost::system::error_code wouldn't work with them, but it does. It just doesn't use one for compatibility with pre-C++11 So just an assumption I failed to update.

Reply:
Scoped!